internal/cephfs/volumeoptions.go

```diff
@@ -478,6 +485,11 @@ func newSnapshotOptionsFromID(ctx context.Context, snapID string, cr *util.Crede
 		return &volOptions, nil, &sid, fmt.Errorf("failed to fetch subvolumegroup list using clusterID (%s): %w", vi.ClusterID, err)
 	}
 
+	err = volOptions.Connect()
```

Comment:
newSnapshotOptionsFromID is also called in CreateVolume. Destroy is already handled there?

Reply:
yes, I think I verified all calls/paths to this function